client/src/Calendar.jsx

```diff
@@ -100,16 +100,16 @@ class Calendar extends React.Component {
     var month1dates = this.props.dates.filter((date) => {
       var adate = new Date(date.date);
       return adate.getMonth() === this.state.month1Date.getMonth();
-    })
+    });
     var month2dates = this.props.dates.filter((date) => {
       var adate = new Date(date.date);
```

Comment:
Curious about the variable 'adate'.
Does it mean arrival date or a temp date variable?